arelle/plugin/loadFromOIM.py

```diff
@@ -3025,16 +3014,10 @@ def validateFinally(val, *args, **kwargs):
                             _("Contexts MUST not contain non-dimensional content: %(contexts)s"),
                             modelObject=contextsWithNonDimContent,
                             contexts=", ".join(sorted(c.id for c in contextsWithNonDimContent)))
-        if contextsWithNonDimContainer:
-            modelXbrl.error("xbrlxe:unexpectedContextContent",
-                            _("Contexts not used for dimensions must not contain content in %(containers)s: %(contexts)s"),
-                            modelObject=contextsWithNonDimContainer,
-                            containers=" or ".join(sorted(containersNotUsedForDimensions)),
-                            contexts=", ".join(sorted(c.id for c in contextsWithNonDimContainer)))
         if contextsWithComplexTypedDimensions:
             modelXbrl.error("xbrlxe:unsupportedComplexTypedDimension",  # this pertains only to xBRL-XML validation (JSON and CSV were checked during loading when loadedFromOIM is True)
                             _("Instance has contexts with complex typed dimensions: %(contexts)s"),
-                            modelObject=contextsWithNonDimContainer,
+                            modelObject=contextsWithComplexTypedDimensions,
```

Comment:
To be sure, this is a tangential bug fix, right?

Reply:
Correct